README.md

```diff
@@ -315,6 +316,10 @@ For `OnPlayChoiceEvent` you must also pass an integer representing the index of
 selected then you should pass `INDEX_NONE` as the index. Note that this will result in no choice statements being executed - the Supertalk player will simply
 continue to the next statement after the current set of choices.
 
+After you created the Supertalk player, you can run `STPlayer.RunScript(YourScript)` where `YourScript` is a `USupertalkScript` asset. 
```

Comment:
Minor grammatical fix: "after you have created..."

I'd also prefer you to use standard C++ syntax here rather than pseudocode: "you can run `USupertalkPlayer::RunScript(USupertalkSctipt* Script)`."